netmap/filter.go

```diff
@@ -13,17 +11,18 @@ const mainFilterName = "*"
 
 // processFilters processes filters and returns error is any of them is invalid.
 func (c *context) processFilters(p PlacementPolicy) error {
-	for i := range p.filters {
-		if err := c.processFilter(p.filters[i], true); err != nil {
-			return fmt.Errorf("process filter #%d (%s): %w", i, p.filters[i].GetName(), err)
+	fs := p.Filters()
```

Comment:
This makes a copy, but I doubt we need it.

Reply:
reverted all